Sources/StreamChatUI/UIConfig+MessageList.swift

```diff
@@ -0,0 +1,97 @@
+            _ChatMessageActionsView<ExtraData>.ActionButton.self
+    }
+
+    struct MessageReactions {
+        public var reactionsBubbleView: _ChatMessageReactionsBubbleView<ExtraData>.Type =
+            _ChatMessageDefaultReactionsBubbleView<ExtraData>.self
+        public var reactionsView: _ChatMessageReactionsView<ExtraData>.Type = _ChatMessageReactionsView<ExtraData>.self
+        public var reactionItemView: _ChatMessageReactionsView<ExtraData>.ItemView.Type =
+            _ChatMessageReactionsView<ExtraData>.ItemView.self
+    }
+
+    struct MessageContentViewSubviews {
+        public var authorAvatarView: ChatAvatarView.Type = ChatAvatarView.self
+        public var bubbleView: _ChatMessageBubbleView<ExtraData>.Type = _ChatMessageBubbleView<ExtraData>.self
+        public var metadataView: _ChatMessageMetadataView<ExtraData>.Type = _ChatMessageMetadataView<ExtraData>.self
+        public var quotedMessageBubbleView: _ChatMessageQuoteBubbleView<ExtraData>.Type = _ChatMessageQuoteBubbleView.self
+        public var attachmentSubviews = MessageAttachmentViewSubviews()
+        public var onlyVisibleForCurrentUserIndicator: ChatMessageOnlyVisibleForCurrentUserIndicator<ExtraData>.Type =
+            ChatMessageOnlyVisibleForCurrentUserIndicator<ExtraData>.self
+        public var threadArrowView: _ChatMessageThreadArrowView<ExtraData>.Type = _ChatMessageThreadArrowView<ExtraData>.self
+        public var threadInfoView: _ChatMessageThreadInfoView<ExtraData>.Type = _ChatMessageThreadInfoView<ExtraData>.self
+        public var threadParticipantAvatarView: ChatAvatarView.Type = ChatAvatarView.self
+        public var errorIndicator: _ChatMessageErrorIndicator<ExtraData>.Type = _ChatMessageErrorIndicator<ExtraData>.self
+        public var linkPreviewView: _ChatMessageLinkPreviewView<ExtraData>.Type = _ChatMessageLinkPreviewView<ExtraData>.self
+    }
+
+    struct MessageAttachmentViewSubviews {
+        public var loadingIndicator: _ChatLoadingIndicator<ExtraData>.Type = _ChatLoadingIndicator<ExtraData>.self
+        public var attachmentsView: _ChatMessageAttachmentsView<ExtraData>.Type = _ChatMessageAttachmentsView<ExtraData>.self
+        // Files
+        public var fileAttachmentListView: _ChatMessageFileAttachmentListView<ExtraData>
+            .Type = _ChatMessageFileAttachmentListView<ExtraData>.self
+        public var fileAttachmentItemView: _ChatMessageFileAttachmentListView<ExtraData>.ItemView.Type =
+            _ChatMessageFileAttachmentListView<ExtraData>.ItemView.self
+        public var fileFallbackIcon = UIImage(named: "generic", in: .streamChatUI)!
+        public var fileIcons = [AttachmentFileType: UIImage](
+            uniqueKeysWithValues: AttachmentFileType.allCases.compactMap {
+                guard let icon = UIImage(named: $0.rawValue, in: .streamChatUI) else { return nil }
+                return ($0, icon)
+            }
+        )
```

Comment:
Isn't this already defined in `documentPreviews`?

Reply:
Okay somehow when recreating the commit this stayed in there. This should definitely not be there but it's quite different, documentPreviews do not have the dictionary key of `AttachmentFileType`... I moved this to `UIConfig+Images` too it's a different case a bit.